course/frontend/arrays/scripts/main.js

```diff
@@ -213,3 +213,80 @@ function flat(arr, newArray) {
 
 flat([1, 2, 3, [4, 5], 6]);
 flat([1, 2, [3, 4, [1, 2, [1, 2, 3]]], 5, 6]);
+
+
+// Exercises 28, 29
+
+/*
+### unionArray
+
+Write a JavaScript program to compute the union of two arrays.
+Sample Data :
+
+    console.log(unionArray([1, 2, 3], [100, 2, 1, 10]));
+    [1, 2, 3, 10, 100]
+
+### removeFalsy
+
+Write a JavaScript function to remove. 'null', '0', '""', 'false', 'undefined' and 'NaN' values from an array.
+
+    Sample array : [NaN, 0, 15, false, -22, '',undefined, 47, null]
+    Expected result : [15, -22, 47]
+
+*/
+
+/** 
+ * Function unionArray 
+ * Returns one numeric merged array without duplicated values.
+ * @param {array} first argument
+ * @param {array} second argument
+ * @returns {array} numeric values.
+ */
+'use strict';
```

Comment:
MOVE 'use strict'; back to each self-invoking function (each one may have it if you apply strict mode to your own code, something that i encourage you always to do). the point on this is that you make strict coding there inside, and do not force outer code to work strictly  (imagine third-party code around, not working in that way). so, in brief, this is totally right:

var func1;
(function() {
  'use strict';
 ... impl func1
})();

var func2;
(function() {
  'use strict';
 ... impl func2
})();